products/fluent-bit.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Fast and Lightweight Log processor and forwarder
+category: app
+tags: kubernetes
```

Comment:
We're using cncf for most products, and reserving the kubernetes tag for products that are very specific to k8s.

```suggestion
tags: cncf
```